src/pages/GovPages/Proposals/index.tsx

```diff
@@ -0,0 +1,183 @@
+import styled, { ThemeContext } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import { Flex, Text } from 'rebass'
+
+import { TYPE } from '../../../theme'
+import { PageWrapper } from '../../Pool/styleds'
+import { useRouter } from '../../../hooks/useRouter'
+
+import { LightCard } from '../../../components/Card'
+import { RowBetween } from '../../../components/Row'
+import { AutoColumn } from '../../../components/Column'
+import CurrencyLogo from '../../../components/CurrencyLogo'
+import { ButtonPrimary, ButtonSecondary } from '../../../components/Button'
+import TabBar from '../../../components/TabBar'
+import Proposals from './Proposals'
+
+// import { Redirect } from 'react-router-dom'
+
+const ContentCard = styled(LightCard)`
+  background: linear-gradient(113.18deg, rgba(255, 255, 255, 0.35) -0.1%, rgba(0, 0, 0, 0) 98.9%),
+    ${({ theme }) => theme.dark1};
+  background-blend-mode: overlay, normal;
+  border-radius: 8px;
+  padding: 25px 20px;
+  position: relative;
+  border: none;
+
+  ::before {
+    content: '';
+    background-image: linear-gradient(180deg, #14131d 0%, rgb(68 65 99 / 50%) 100%);
+    top: -1px;
+    left: -1px;
+    bottom: -1px;
+    right: -1px;
+    position: absolute;
+    z-index: -1;
+    border-radius: 8px;
+  }
+`
+
+const TitleRow = styled(RowBetween)`
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    flex-wrap: wrap;
+    gap: 12px;
+    width: 100%;    
+  `};
+`
+
+const ContentTitle = styled(TYPE.main)`
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 17px;
+  color: ${({ theme }) => theme.purple2};
+`
+
+const ResponsiveButtonPrimary = styled(ButtonPrimary)`
+  width: fit-content;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    width: 100%;
+  `};
+`
+
+const ResponsiveButtonSecondary = styled(ButtonSecondary)`
+  width: fit-content;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    width: 100%;
+  `};
+  padding: 9px 21px;
+  background: linear-gradient(141.72deg, rgba(255, 255, 255, 0.55) -11.46%, rgba(0, 0, 0, 0) 155.17%),
+    ${({ theme }) => theme.dark1};
+  background-blend-mode: overlay;
+  border: 1px solid #28263f;
+`
+
+export default function GovernanceProposals() {
+  const { t } = useTranslation()
+  const theme = useContext(ThemeContext)
+  const router = useRouter()
+  const pairName = t(router.query.asset) + '/' + t(router.query.pair)
+  const currency = router.location.state?.currency
+  const currency1 = router.location.state?.currency1
+
+  // if (currency === undefined || currency1 === undefined) {
+  //   return <Redirect to="/" />
+  // }
+
+  return (
+    <PageWrapper>
+      <AutoColumn gap="lg" justify="center">
+        <AutoColumn gap="lg" style={{ width: '100%' }}>
+          <TitleRow marginBottom="1rem" padding="0">
+            <Flex alignItems="center">
+              <a href="#/governance">
+                <TYPE.mediumHeader color={theme.text4} lineHeight="24.38px" fontWeight={400}>
```

Comment:
In these cases, without using the `useContext` hook, you can write directly the name of the key you want to use in the theme color (in this case, `text4`).